front/public/locales/en/errors.json

```
@@ -208,7 +208,8 @@
       "RollingStockNotFound": "Rolling stock '{{rolling_stock_id}}' does not exist",
       "TowedRollingStockNotFound": "Towed rolling stock {towed_rolling_stock_id} does not exist",
       "TrainSimulationFail": "Train simulation failed",
-      "TimetableNotFound": "Timetable '{{timetable_id}}' does not exist"
+      "TimetableNotFound": "Timetable '{{timetable_id}}' does not exist",
+      "InvalidRequest": "Request validation failed"
```

I think we should be consistent and more direct here too:
```suggestion
      "InvalidRequest": "Invalid request"
```